tools/api-markdown-documenter/CHANGELOG.md

```diff
@@ -8,24 +8,6 @@
 
 Combines the separate "config" property bag parameters into a single "options" property bag for simplicity.
 
-#### Type-renames
-
--   `ConfigurationBase` -> `LoggingConfiguration`
--   `RenderDocumentAsHtmlConfig` -> `RenderDocumentAsHtmlConfiguration`
--   `RenderHtmlConfig` -> `RenderHtmlConfiguration`
--   `ToHtmlConfig` -> `ToHtmlConfiguration`
-
-#### Configuration properties made `readonly`
-
--   `ApiItemTransformationConfiguration`
--   `ApiItemTransformationOptions`
--   `DocumentationSuiteOptions`
--   `HtmlRenderer.RenderHtmlConfig`
--   `LintApiModelConfiguration`
--   `MarkdownRenderer.Renderers`
--   `MarkdownRenderer.RenderContext`
--   `ToHtmlTransformations`
-
 ##### Example
```

Comment:
This was supposed to be under the earlier heading, but was accidentally moved in a previous PR.